File: client/src/app/models/notebook/note.ts
import { BaseModel } from "../base/base-model";

export class Note extends BaseModel {
  public title  = "";
  public content  = "";
  public order = 0;
  public categoryId  = "";
  public categoryName  = "";
  public isPublic = false;
  public isPinned = false;
  public originBackgroundColor = "#fff";
  public backgroundColor = "#fff";
  public ownerId = "0";
}

File: server/Src/Services/Open/OpenVN.Domain/Entities/Notebook/Note.cs
namespace OpenVN.Domain
{
    public class Note : PersonalizedEntity
    {
        public string Title { get; set; }

        public string Content { get; set; }

        public int Order { get; set; }

        public long CategoryId { get; set; }

        public bool IsPublic { get; set; }

        public bool IsPinned { get; set; }

        public string BackgroundColor { get; set; }
    }
}
